tests/test_receivers.py

```diff
@@ -65,3 +67,26 @@ def test_build_cache_with_no_collections(app):
 
         # now the collection 'mycoll' should appear in `record0` of A!
         assert record0['_collections'] == ['mycoll']
+
+
+def test_cache_can_be_imported_from_a_config_variable(app):
+    InvenioCollections(app)
+    app.config['COLLECTIONS_CACHE'] = 'werkzeug.contrib.cache:SimpleCache'
```

Comment:
See e.g. https://github.com/inveniosoftware/invenio-indexer/blob/f8a30f1a2631a2b8e9733d3149fe5c71eedc8980/tests/test_invenio_bulkindexer.py#L68 and https://github.com/inveniosoftware/invenio-indexer/blob/f8a30f1a2631a2b8e9733d3149fe5c71eedc8980/tests/test_invenio_bulkindexer.py#L40-L41